assets/js/blocks/mini-cart/block.json

```diff
@@ -47,19 +42,19 @@
 			"default": "hidden"
 		},
 		"priceColor": {
-			"type": "string"
+			"type": "object"
 		},
 		"priceColorValue": {
```

Comment:
Annoyingly with the current method of maintaining existing blocks that have custom colors we need to keep these values in the `block.json` file.